src/data/input_datetime.ts

```diff
@@ -0,0 +1,15 @@
+import { HomeAssistant } from "../types";
+
+export const setValue = (
+  hass: HomeAssistant,
+  entity: string,
+  time: string | null = null,
```

Comment:
Instead of this, just add a values object.

```ts
options: {
  time?: string,
  date?: string
}
```

JS fun: keys with the value `undefined` will be omitted in `JSON.stringify` so no need for `null` checks`

Reply:
Could have sworn I tried that... or perhaps that was just `null`...